2020/11/index.js

```diff
@@ -1,3 +1,223 @@
+const { promises: fs } = require('fs');
+
+async function getInput() {
+  return input = (await fs.readFile(`${__dirname}/input.txt`))
+    .toString()
+    .trim()
+    .split('\n')
+    .map(row => row.split(''));
+}
+
+function p1NumAdjOccupiedSeats(input, row, column) {
```

Comment:
Oof, I hope you generated this code! I think my solution is very succinct and easier to understand [here](https://github.com/faizanminhas/advent-of-code-2020/pull/4/files#diff-2591cbae71189f9147ae87772a1bd929a92ac64f31d511b8700c99388786d03aR7)

Reply:
Lol I had major struggles figuring how to skip [0, 0]. Didn't know the OR and NOT 0 check was the way to go.